frontend/packages/shared/src/components/GiteaHeader/VersionControlButtons/components/ShareChangesPopover/ShareChangesPopover.tsx

```diff
@@ -49,40 +54,46 @@ export const ShareChangesPopover = () => {
     if (!hasPushRights) {
       return t('sync_header.sharing_changes_no_access');
     }
-    if (hasMergeConflict) {
-      return t('sync_header.merge_conflict_title');
-    }
     return t('sync_header.changes_to_share');
   };
 
   return (
-    <StudioPopover open={popoverOpen} onClose={handleClosePopover} placement='bottom-end'>
-      <StudioPopover.Trigger asChild>
-        <StudioButton
-          color='inverted'
-          size='small'
-          variant='tertiary'
-          onClick={handleOpenPopover}
-          disabled={!hasPushRights || hasMergeConflict}
-          title={renderCorrectTitle()}
-          icon={hasMergeConflict ? <XMarkIcon /> : <UploadIcon />}
-        >
-          {hasMergeConflict ? t('sync_header.merge_conflict') : t('sync_header.changes_to_share')}
-          {displayNotification && <Notification numChanges={1} />}
-        </StudioButton>
-      </StudioPopover.Trigger>
-      <StudioPopover.Content
-        className={fetchCompleted ? classes.popoverContentCenter : classes.popoverContent}
+    <div>
+      <StudioButton
+        color='inverted'
+        size='small'
+        variant='tertiary'
+        onClick={handleOpenPopover}
+        disabled={!hasPushRights || hasMergeConflict}
```

Comment:
If I understand correctly from your second bullet point in the PR description, we can remove `hasMergeConflict` here.
(And maybe also the logic for this in `VersionControlButtonsContext` - or maybe not if it used elsewhere?

Reply:
Hm, no I dont think so, because this bool will disable the original button if there is any mergeconflict. But the original code, which I removed, changed the text to `Mergekonflikt` and the symbol. 
But this part is only a suggestion from my side. Since I felt it was inconsistent that the button-text changed. If there is a merge conflict you will get a dedicated page telling you there is a merge conflict so it felt unnecessary to change the text as well 🤔

Reply:
ah, I see 🙂  Yeah keep it there ^^ 
![image](https://github.com/Altinn/altinn-studio/assets/74791975/4e241f2b-1edc-413c-8be2-f0cbfd416b36)